source/tutorials/security/trusty-enabling/trusty-enabling.rst

```diff
@@ -0,0 +1,73 @@
+.. _trusty-enabling:
+
+How to Enable or Disable Trusty for Debugging
+#############################################
+
+`Trusty <https://01.org/trusty-ia>`_ is a mandatory component since Android Oreo dessert onwards. It provides the hardware backed security services such as the keymaster, gatekeeper, crypto etc, which are required for applications such as data storage encryption etc.
+
+How to enable Trusty
+====================
+
+Project |C| supports Trusty. To enable Trusty explicitly, update the config file *device/intel/project-celadon/$<lunch-target>/mixins.spec* in the device config project '*device-androidia*' as follows:
+
+.. code-block:: none
+    :emphasize-lines: 6
+
+    ..
+    [groups]
+    device-specific: celadon
+    treble: true
+    ..
+    trusty: true(ref_target=$<target name>)
+
+The following parameters are valid options within the brackets in the previous configuration setting:
+
+* **ref_target** : ``mandatory``
+    This option defines the reference target device.
+
+    For project |C|, *<target name>* can be any of below values
+
+        *project-celadon_apl* : For IVI usage running on Intel Apollo Lake family
+
+        *project-celadon_kbl* : For IVI usage running on Intel Kabe Lake Core processor
+
+        *project-celadon_64* : For tablet usage running on for Commercial Kabe Lake and Apollo Lake NUCs
+
+* **enable_hw_sec** : ``optional``
+    This flag defines the Trusty feature is backed by hardware or software. By setting this option to true, the security services are provided by the Trusty, or the software backed security services are used.
```

Comment:
Suggested rewrite: 
This flag indicates the Trusty feature is backed by hardware or software. By setting this option to true, the security services are provided by the Trusty, or the software-backed security services are used.